rtc2git.cli.extension/src/to/rtc/cli/migrate/git/GitMigrator.java

```diff
@@ -212,6 +212,7 @@ String getWorkItemTexts(List<WorkItem> workItems) {
 
 	private void gitCommit(PersonIdent ident, String comment) {
 		try {
+			initRootGitignore(rootDir);
```

Comment:
It is intentional. Actually, the initRootGitignore rebuilds the content of the root .gitignore file based on the defaults and the .jazzignore in the root if it exists. It is easier to handle the changes, if any, here than in handleJazzignores function. In case the .jazzignore is deleted it results in a change of the .gitignore but in handleJazzignores function, you cannot add .gitignore to toAdd list as it was called with toRestore and can return files to be added to the toRemove list only.